docs/source/devguide/contributors.rst

```diff
@@ -25,20 +25,16 @@ else may have time to finalize for approval even if you are unable.
 
 Contributions generally follow the process of:
 
-1. `Submit a GitHub Issue`_ for a bug fix or feature request.
-2. `Branch off a given Issue`_ in order to make the required changes
-3. `Create a github Pull Request`_ in order for your changes to be reviewed
-   prior to being merged to the integration branch
+1. Submit a GitHub Issue for a bug fix or feature request. :ref:`create_issue`
```

Comment:
Are these ":ref:" constructs supposed to link somewhere?  It doesn't appear that they do.

Reply:
Only when you actually build the documentation - they are sphinx directives.  We went back and forth on github native RST rendering (which do not link in HTML documentation), and sphinx directives (which are not clickable from within github).  Thoughts?

Reply:
I suppose it's probably better to have it in the built documentation if you only have one.  Is there anyway to have both links without it looking weird?

Reply:
I think we will have to address this through a separate issue - I am going to resolve this for now.